libs/application/templates/accident-notification/src/fields/ApplicationStatus/StatusStep/index.tsx

```diff
@@ -71,6 +71,7 @@ export const StatusStep: FC<ReviewSectionProps> = ({
           justifyContent="spaceBetween"
           background="roseTinted100"
           padding={4}
+          className={cn(styles.bottomBorderRadius)}
```

Comment:
Dont think you need cn as we arent joining classnames together here, do you?

Reply:
Propably not, removed 😎